Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -922,10 +932,17 @@ class SettingsTableViewController: UITableViewController {
 
     override func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
         //make account/sign-in and close private tabs rows taller, as per design specs
-        if ((indexPath.section == 0 && indexPath.row == 0) ||
-            (indexPath.section == 2 && indexPath.row == 1)) {
-            return 64
+        if #available(iOS 9, *) {
+            if ((indexPath.section == 0 && indexPath.row == 0) ||
+                (indexPath.section == 2 && indexPath.row == 1)) {
+                return 64
+            }
+        } else {
+            if indexPath.section == 0 && indexPath.row == 0 {
+                return 64
+            }
```

Comment:
I've simplified the if statement logic. I think eventually we'll probably want to have each setting tell the delegate method what height it should be instead of hard coding the indexes here. It can be fragile since we're usually adding/removing settings options frequently. I think whenever we revisit the IA of settings we should add this in.